docs/Developers/Git_Basics.rst

```diff
@@ -245,6 +245,12 @@ the desired branch:
     $ git fetch origin
     $ git rebase origin/master
 
+.. note::
+
+    ``origin`` is the remote of https://github.com/coala/coala
+    or https://github.com/coala/coala-bears. It is not the remote
+    of your fork.
```

Comment:
I agree with you. I always have `upstream` as the original and `origin` as my fork. But I was not around when the original guide was made and I don't know if there's a specific reason for having it this way, so I'd like to wait for someone else to chime in before we change this.